app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
I like that solution @tomas-stefano - my only other thought/concern is that `LOW_SALARY_THRESHOLDS` are linked to the `SALARY_FREQUENCIES`.
i.e. if for whatever reason we decide to have one more or one less frequency, unlikely as it is, the thresholds will also need to have one added/removed. 
Suggests to me they might need to be in the same yaml file?